commands/reverseintake.py

```diff
@@ -0,0 +1,25 @@
+from commands2 import CommandBase
+
+from subsystems.intakesubsystem import IntakeSubsystem
+
+
+class ReverseIntake(CommandBase):
+    def __init__(self, intake: IntakeSubsystem) -> None:
+        CommandBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intake = intake
+        self.addRequirements(intake)
+
+    def initialize(self) -> None:
+        print(f"Command: {self.getName()}")
+        self.intake.reverseIntake()
+
+        if self.intake.isIntakeDeployed():
+            self.intake.runIntake(self.intake.intakeReversed)
+
+    # pylint: disable=unused-argument,no-self-use
```

Comment:
Change this to 
`# pylint: disable-next=unused-argument,no-self-use`

And then add to the following above isFinished
`# pylint: disable-next=no-self-use`

disable-next disables the next line.  Which is better than disable which disables until the end of the block or the end of the file.